### PR TITLE
Copter: 4.1.2-rc1 beta release

### PR DESCRIPTION
**Pull request by rmackay9**
This is the PR to update the Copter-4.1 branch for the 4.1.2-rc1 beta release.  This is "double rebased" on the Plane-4.1 branch making it pretty much equivalent to the Plane-4.1.3 release with these additions:

- AP_Motors: change H_COLL_HOVER to H_COL_HOVER https://github.com/ArduPilot/ardupilot/pull/19295
- Copter: surface tracking timeout fix https://github.com/ArduPilot/ardupilot/pull/19207
- Add FETtec OneWire (ESC telemetry) support https://github.com/ArduPilot/ardupilot/pull/18631

### CODE REVIEW

**Comment by tridge** [comment]
@rmackay9 we need to check if adding FETTec support takes us over the flash limit - we still use the old compiler for stable releases

**Comment by amilcarlucas** [comment]
All CI tests pass. :)

**Comment by rmackay9** [comment]
@amilcarlucas,

I suspect that the CI tests always use the latest compiler regardless of version so we may need to more specifically test that it builds using the correct compiler.

File: ArduCopter/ReleaseNotes.txt [hunk 1]
```diff
@@ -1,5 +1,17 @@
 ArduPilot Copter Release Notes:
 ------------------------------------------------------------------
+Copter 4.1.2-rc1 22-Nov-2021
+Changes from 4.1.1
+1) CAN_Dn_UC_OPTION param added to help resolve DroneCAN DNA conflicts
+2) Durandal with alternative ICM-20602 IMU
+3) OBAL autopilot support (Open Board Architecture for Linux)
+4) Bug Fixes
```
Comment (amilcarlucas) [inline ArduCopter/ReleaseNotes.txt:8]:
```suggestion
4) Added support for FETtec One wire protocol - a single UART serial ESC with telemetry
5) Bug Fixes
```

**Review by amilcarlucas — CHANGES_REQUESTED** [review]

**Comment by rmackay9** [comment]
@andyp1per suggests we should add these bug fixes as well

- https://github.com/ArduPilot/ardupilot/pull/19166
- https://github.com/ArduPilot/ardupilot/pull/19246

File: ArduCopter/ReleaseNotes.txt (hunk 1 above)
Comment (rmackay9) [reply ArduCopter/ReleaseNotes.txt:8]: added, thanks

**Comment by rmackay9** [comment]
@tridge has offered to check that this fits with the older compiler that we use for 4.1 so once that is confirmed we will merge this.

**Pull request merged**